test/evergreen_develop.yml

```diff
@@ -75,6 +78,7 @@ buildvariants:
     - name: ".checkpoint-perf"
     - name: ".evict-perf"
     - name: ".log-perf"
+    - name: ".chunk-cache-perf"
```

Comment:
Are there any additional variants we want to do perf tests for or just the ubuntu2004 ones?

Reply:
These Ubuntu 20.04 variants are what end up on the performance charts, so I picked these two for automatically running it. So my vote is no, but I'm happy to hear arguments otherwise.

Reply:
Fair call - we can always add them later if we are interested in those results.